src/theme.ts

```diff
@@ -78,6 +91,11 @@ export default createTheme({
       outlined: {
         backgroundColor: 'white',
         border: '1px solid #8891AA',
+        [defaultTheme.breakpoints.down('md')]: {
```

Comment:
Should this be `defaultTheme.breakpoints.down(BREAKPOINTS.values.md`?

Reply:
actually ignore this comment. there must have been something going on with the hot-reloading last night